src/Tribe/Template.php

```diff
@@ -402,27 +402,27 @@ protected function get_template_public_path( $base ) {
 	 * @return array
 	 */
 	protected function get_template_path_list() {
-		$folders = array();
+		$folders = [];
 
 		// Only look into public folders if we tell to use folders
 		if ( $this->template_folder_lookup ) {
-			$folders[] = array(
+			$folders[] = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 5f6a611a2c281f27762d2f6909201050</sub>